index.html

```diff
@@ -41,10 +42,148 @@ <h2 class="secondinfo">I’m a software developer</h2>
     </ul>
     
   </section>   
-        
-   
+  
+  <section class="intro-works">
+
+    <button class="deco-button"><img class="img-arrow" src="images/arrow-dropdown.jpg" alt="arrow dropdown"></button>
+    <h1 class="works-text">My Recent Works</h1>
+    <div class="indicator"></div>
+
+  </section>
+
+
+<!--Grid container with all 6 work templates--> 
+
+  <div class="grid-container">
+
+    <section class="works-template work1">
+      <div class="work-data">
+        <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+        <div class="tag-holder">
```

Comment:
- [ ] Kindly replace `div ` with a `ul` and use the `li` tag to list the technologies used in the project.